Gtest/AudioFixture.cpp

```
@@ -0,0 +1,714 @@
+/*
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dlfcn.h>
+#include <errno.h>
+#include <limits.h>
+#include <unistd.h>
+#include <log/log.h>
+#include "AudioFixture.h"
+
+#define MODULE_NAME "/system/vendor/lib64/audio.primary.cic_cloud.so"
+#define STUB_OUTPUT_BUFFER_MILLISECONDS 10
+
+using namespace std;
+using namespace std::chrono_literals;
+
+const hw_module_t *mod;
+audio_hw_device_t* halDevice = nullptr;
+
+void AudioFixture::runAudioInputThread() {
+    mAudioClient.startDummyStreamerForInput();
+}
+
+
+void AudioFixture::runAudioOutputThread() {
+    mAudioClient.startDummyStreamerForOutput();
+}
+
+int load(const char *id,
+        const struct hw_module_t **pHmi)
+{
+    int status = -EINVAL;
+    void *handle = NULL;
+    struct hw_module_t *hmi = NULL;
+    const char *sym = HAL_MODULE_INFO_SYM_AS_STR;
+
+    handle = dlopen(MODULE_NAME, RTLD_NOW);
+    if (handle == NULL) {
+        char const *err_str = dlerror();
+        ALOGE("load: module=%s, %s", MODULE_NAME, err_str ? err_str : "unknown");
+        status = -EINVAL;
+        goto done;
+    }
+    ALOGE("load: module=%s success", MODULE_NAME);
```

Can you change the non-failure logs to info or debug rather than using ALOGE ?